app/components/GuideReviewForm/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Select from 'components/Select';
+
+import PageSection from 'components/PageSection';
+import BorderBottomDiv from 'components/shared/BorderBottomDiv';
+import Stars from 'components/Stars';
+
+class GuideReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.state = {
+      isGuided: props.isGuided,
+    };
+  }
+
+  componentWillMount() {
+    if(this.props.isGuided) {
+      this.props.getGuideList();  // lifting up state: API call is handled by WriteReviewPage container
+    }
+  }
+
+  shouldComponentUpdate(nextProps, nextState) {
+    // Update if...
+    const doUpdate = ((nextState.isGuided !== this.state.isGuided) //...switching between yes/no options (affects disabled status for Select component)
+      || (nextProps.guideOptions.length !== this.props.guideOptions.length) //...or guide list is updated (affects Select component's options)
+      || (nextProps.selectedGuideId !== this.props.selectedGuideId) //...or a guide is selected (affects Select component and disabled status for Stars component)
+    );
+
+    return doUpdate;
+  }
+
+  componentWillUpdate(nextProps) {
+    // Do another API call if, somehow, the options array's length changes
+    if(this.props.guideOptions.length && nextProps.guideOptions.length !== this.props.guideOptions.length) {
+      console.log('componentWillUpdate newAPIcall:', nextProps.guideOptions);
+      this.props.getGuideList();  // lifting up state: API call is handled by WriteReviewPage container
+    }
+  }
+
+  handleButtonClick(e) {
+    // There are edge cases where clicking on the edge of a button can give undefined value
+    const isGuided = e.target.firstChild.value ? e.target.firstChild.value === "guided" : this.state.isGuided;
+
+    this.setState({
+      isGuided: isGuided,
+    });
+
+    // If "No" is selected, clear the Select component (this disables the rating as well)
+    if(!isGuided) {
+      this.props.handleSelectGuide(null);
+    }
+    else if(isGuided && !this.props.guideOptions.length) {
+      // Do API call if "yes" is selected and options array is currently empty
+      this.props.getGuideList();
+    }
+  }
+
+  render() {
+    return (
+      <PageSection title="Was the trip guided?">
+        {/* Yes/No buttons */}
+        <div className="btn-group-lg btn-group-toggle d-flex mx-3" data-toggle="buttons" onClick={this.handleButtonClick}>
+          <label className="btn btn-block mr-3 btn-success">
+            <input type="radio" value="guided" autoComplete="off"/>
```

Comment:
good call on radio buttons, I find the implementation visually confusing though, can you change them just to use the primary/secondary theme colors? Also make them a bit smaller. We'll likely adjust later anyway but the red/green thing seems off

Reply:
Yeah, I went with the default of what I grabbed to start with, and there's no specifics on the buttons on the Zeplin screens (just blank rectangles). I went with bigger buttons because I rationalize that they're more helpful to be big on touch screens where fingers can be imprecise. Easy to change, though.

Reply:
I used the outline button-color classes, so it's very obvious when one is selected now.